content/en/docs/concepts/instrumentation/zero-code.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Zero-Code
+description: >-
+  Learn how to add observability to an application without the need to write more
+  code
+weight: 10
+aliases: [automatic]
+---
+
+As [ops](/docs/getting-started/ops/) you might want to add observability to one
+or more applications without having to edit the source, probably because you can't
+or you're not allowed to. In other cases, you might be looking for a way to quickly
+gain some OTel observability for a service before diving into learning how to use
+the OpenTelemetry API & SDK for
+[code-based instrumentation](/docs/concepts/instrumentation/code-based).
+
+In all the previous cases, you can leverage language specific zero-code
+solutions, which use mechanisms like bytecode instrumentation, monkey patching,
+or eBPF to inject calls to the OpenTelemetry API and SDK into your application.
+
+While the underlying mechanism depends on the language, zero-code
+instrumentation adds the OpenTelemetry API and SDK capabilities to your
+application with almost no effort. In addition, you can use the same process
+to add a set of
+[Instrumentation Libraries](/docs/concepts/instrumentation/libraries) and
+[exporter](/docs/concepts/components/#exporters) dependencies.
```

Comment:
```suggestion
Zero-code instrumentation adds the OpenTelemetry API and SDK capabilities to your
application typically as an agent or agent-like installation. The specific mechanisms
involved may differ by language, ranging from bytecode manipulation, monkey patching,
or eBPF to inject calls to the OpenTelemetry API and SDK into your application.

Typically, zero-code instrumentation adds instrumentation for the libraries you're using.
This means that requests and responses, database calls, message queue calls, and so forth
are what are instrumented. Your application's code, however, is not typically instrumented.
To instrument your code, you'll need to use [code-based instrumentation](/docs/concepts/instrumentation/code-based)

Additionally, zero-code instrumentation lets you configure the [Instrumentation Libraries](/docs/concepts/instrumentation/libraries) and [exporters](/docs/concepts/components/#exporters) loaded.
```

I ended up rewriting this. I think it flows better when we talk about what it typically does, how it does it, and hint a bit at configuration before diving into the config paragraph.

Reply:
thank you!

Reply:
applied